Sources/hostmgr-helper/ManagedVirtualMachine.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+@preconcurrency import Virtualization
```

Comment:
TIL `@preconcurrency`. From [the docs](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#preconcurrency):

> Apply this attribute to a declaration, to suppress strict concurrency checking.
> [...]
> On an import declaration, this attribute reduces the strictness of concurrency checking for code that uses types from the imported module. Specifically, types from the imported module that aren’t explicitly marked as nonsendable can be used in a context that requires sendable types.